src/current/cockroachcloud/costs.md

```diff
@@ -163,17 +177,29 @@ Customers will not be charged for data transfer for CockroachDB {{ site.data.pro
 
-The usage data is based on the volume of data moved (per GiB) and the cloud provider’s line item for each type of data transfer, which CockroachDB categorizes as **Same-region data transfer**, **Cross-Region data transfer**, or **Public Internet data transfer**.
+The usage data is based on the volume of data moved (per GiB) and the cloud provider’s identified data transfer type, which CockroachDB categorizes as **Same-region data transfer**, **Cross-Region data transfer**, or **Public Internet data transfer**. Each category is reflected as a line item on your invoice.
 
 Cloud providers may vary in how they calculate each category. We recommend consulting the provider’s documentation for details (i.e. [AWS](https://aws.amazon.com/ec2/pricing/on-demand/), [GCP](https://cloud.google.com/vpc/network-pricing), or [Azure](https://azure.microsoft.com/en-us/pricing/details/bandwidth/)).
 
 #### Same-region
 
-A minimum production grade CockroachDB Advanced deployment consists of three nodes or more placed across three Availability Zones (AZs) within the selected region. When data moves between these AZs within a single region, this is considered same-region data transfer.
+There are no charges for same-region data transfer between CockroachDB cluster nodes. Data egressing the cluster to endpoints in the same region for backups or changefeeds, or using private connectivity, is charged the cloud provider's list price for same-region data transfer.
 
 #### Cross-region
 
-Similar to same-region data transfer , when data moves between CockroachDB nodes across multiple regions, we get cross-region charges (also known as inter-region charges). Typically, data transfer costs across regions are higher than data transfer costs across AZs (data centers) in a single region.
+When data moves between CockroachDB nodes that are in different regions, or from a CockroachDB node to another endpoint in a different region, it is metered as cross-region data transfer. 
+
+Cross-region data transfer includes:
+
+- Data transfer required to support queries that involve lookups on nodes in another region.  
+- CockroachDB replication across nodes that are in different regions.  
+- Data egress from the CockroachDB Cloud cluster via supported [private connectivity](https://www.cockroachlabs.com/docs/cockroachcloud/connect-to-your-cluster#establish-private-connectivity) services to a private endpoint in another region.  
+- [Managed backup](https://www.cockroachlabs.com/docs/cockroachcloud/backup-and-restore-overview#managed-backups) and [Self-managed backup](https://www.cockroachlabs.com/docs/cockroachcloud/take-and-restore-self-managed-backups) data transfer to another region.  
+- Change Data Capture (changefeed) data transfer to another region.
+
+Customers are charged the cloud service provider’s (CSP) list price for metered cross-region data transfer.
```

Comment:
```suggestion
Cross-region data transfer is charged at the cloud provider’s list price for metered cross-region data transfer.
```
Is it worth linking to the official cloud provider rates here instead of at the end of the section?